package.json

```diff
@@ -30,6 +34,8 @@
     "@babel/core": "^7.4.0",
     "@babel/preset-env": "^7.4.2",
     "@babel/preset-react": "^7.0.0",
+    "acorn": "^6.1.1",
+    "acorn-dynamic-import": "^4.0.0",
```

Comment:
What are you using this package for?
If this is an unneeded package, remember to uninstall :)

Reply:
Thanks Andrew, these packages were there from when I got the service, so I left them, not knowing how they interacted with Daniel's code. I will look into where it is being used and remove if possible. Best, Uma